### ISSUE TITLE
Not working: i get an invalid request token after dropping a file

### ISSUE THREAD

**Issue by stef59100** [heimrichhannot/contao-multifileupload-bundle#11]
I get a warning message "invalid token" message. Any idea ?

**Comment by koertho** [comment]
@stef59100 Hi, is the error still present in the latest version? Please always provide contao version and bundle version.

**Comment by stef59100** [comment]
Hi, i Guess yes. I installed it on contain 4.9 managed version when I got
this error.

Stéphane Joos
http ://stephanejoos.eu
0630045688

**Comment by stef59100** [comment]
Do you need a screenshot  ?

Stéphane Joos
http ://stephanejoos.eu
0630045688

Le mar. 21 avr. 2020 à 08:54, Stéphane Joos <stephane.joos@gmail.com> a
écrit :

> Hi, i Guess yes. I installed it on contain 4.9 managed version when I got
> this error.
>
> Stéphane Joos
> http ://stephanejoos.eu
> 0630045688
>

**Comment by koertho** [comment]
@stef59100 Yeah, a screenshot could help. Are you using multi file upload in form generator?

Please be aware when replying with email, you publish your phone number and web address in the issue. I have already deleted this informations in the last comment, but it would be better to answer directly on github or delete this informations before replying.

**Comment by stef59100** [comment]
Yes i use it in the form generator
Maybe it's a question of access rights of the target folder (set to 755)

Stéphane Joos
0630045688
www.stephanejoos.eu

**Comment by koertho** [comment]
@stef59100 Hi, today I stumbled on the same error (in another context) and fixed it in version 1.3.1. I don't tried it in the form generator, but maybe this works now too!
